tools/clang/test/CodeGenSPIRV/hull.pcf.primitive-id.hlsl

```diff
@@ -0,0 +1,47 @@
+// Run: %dxc -T hs_6_0 -E main
+
+#include "bezier_common_hull.hlsl"
+
+// Test: PCF takes the PrimitiveID
+// Note that in this test, the main entry point has also taken the PrimitiveID as input.
+
+
+// CHECK: OpEntryPoint TessellationControl %main "main" {{%\w+}} {{%\w+}} %gl_PrimitiveID {{%\w+}}
+
+// CHECK: OpDecorate %gl_PrimitiveID BuiltIn PrimitiveId
+
+// CHECK:     [[fType:%\d+]] = OpTypeFunction %HS_CONSTANT_DATA_OUTPUT %_ptr_Function_uint
+// CHECK:    %gl_PrimitiveID = OpVariable %_ptr_Input_uint Input
+// CHECK: %param_var_PatchID = OpVariable %_ptr_Function_uint Function
```

Comment:
I'd suggest to add `// CHECK-LABEL: %main = OpFunction` or some comments to make it clear the following is in the wrapper.